### PR TITLE
Frogger game

### PR DESCRIPTION
**Pull request by Ant-C-tech**
### Frogger game

[Demo](https://ant-c-tech.github.io/frontend-nanodegree-arcade-game/)
[Code base](https://github.com/Ant-C-tech/frontend-nanodegree-arcade-game/blob/frogger-game-solution/js/app.js)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by OleksiyRudenko** [comment]
@Ant-C-tech did you test your code? Demo doesn't seem to work

**Comment by Ant-C-tech** [comment]
I think it need some time for full load. Now it work, sorry.

File: submissions/Ant-C-tech/frogger-game/app.js [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+        this.x = this.lastX;
+    };
+    if (this.y > PLAYER_START_Y) {
+        this.y = this.lastY;
+    };
+};
+
+
+let allEnemies = [];
+for (let index = 0; index < _getRandomIntInclusive(ENEMY_MIN, ENEMY_MAX); index++) {
```
Comment (OleksiyRudenko) [inline submissions/Ant-C-tech/frogger-game/app.js:102]: `_getRandomIntInclusive` in `index < _getRandomIntInclusive(ENEMY_MIN, ENEMY_MAX)` is called at every loop iteration. Not critical here but may cause issues in general case.

File: submissions/Ant-C-tech/frogger-game/app.js [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+    let bugY;
+    if (index < 3) {
+        bugY = (FIELD_TOP_BORDER + index * FIELD_CELL_HEIGHT);
+    } else {
+        bugY = (FIELD_TOP_BORDER + (index % 3) * FIELD_CELL_HEIGHT);
+    };
```
Comment (OleksiyRudenko) [inline submissions/Ant-C-tech/frogger-game/app.js:108]: How about ternary operator here? 
Besides `0 % 3 === 0`, `1 % 3 === 1` and `2 % 3 === 2`

**Review by OleksiyRudenko — CHANGES_REQUESTED** [review]
@Ant-C-tech a really good job!
Let's polish it.

**Comment by Ant-C-tech** [comment]
Thank you for your review. I am not sure that I properly changed my code according your notice "How about ternary operator here?". It seems to me now it works without conditional construction.
Regards.

**Review by OleksiyRudenko — APPROVED** [review]
@Ant-C-tech great job done!

**Comment by Ant-C-tech** [comment]
Thanks for your review and for the time that your spent to my code!

**Comment by OleksiyRudenko** [comment]
@Ant-C-tech my pleasure. Hope it helped you to improve

**Pull request merged**